dotcom-rendering/src/model/article-schema.json

```diff
@@ -4168,6 +4223,19 @@
                 "sponsorName"
             ]
         },
+        "OnwardsType": {
+            "enum": [
+                "curated-content",
```

Comment:
Are these arbitrary strings or are they matched elsewhere/in CAPI? Curious to know how easy it is to add to this as necessary.

Reply:
This type was [added in this PR](https://github.com/guardian/dotcom-rendering/pull/5550). It's arbitrary in the sense that you can add anything to it but the type constrains what can be sent by Frontend